docs/writing-docs/docs-page.md

```diff
@@ -48,65 +47,66 @@ DocsPage has the concept of a "primary" component that is defined by the `compon
 
 Subcomponent `ArgsTables` will show up in a tabbed interface along with the primary component. The tab titles will correspond to the keys of the subcomponents object.
 
-If you want to organize your documentation differently for component groups, we recommend using MDX. It gives you complete control over display and supports any configuration.
+If you want to organize your documentation differently for component groups, we recommend using MDX. It gives you complete control over how it's displayed and supports any configuration.
 
 ## Replacing DocsPage
 
-Replace DocsPage template with your own for the entire Storybook, a specific component, or a specific story.
-
-Override the `docs.page` [parameter](../writing-stories/parameters.md):
-
-- With null to remove docs.
-- With MDX docs.
-- With a custom component
+Replace the DocsPage template with your own to customize its contents.
 
-### Story-level
-
-Override the `docs.page` [parameter](../writing-stories/parameters.md#story-parameters) in the story definition.
+### With null to remove docs
+ 
+Override the `docs.page` [parameter](../writing-stories/parameters.md) with `null` to remove its contents.
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/button-story-disable-docspage.js.mdx',
+    'common/button-story-disable-docspage-component.js.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-### Component-level
+### With MDX documentation
 
-Override the `docs.page` [parameter](../writing-stories/parameters.md#component-parameters) in the default export of the story file.
+Write your documentation in MDX and update the `docs.page` [parameter](../writing-stories/parameters.md) to display it.
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/button-story-disable-docspage-component.js.mdx',
+    'common/custom-docs-page.mdx.mdx',
+    'common/button-story-docspage-with-mdx.js.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-### Global-level
+### With a custom component
 
-Override the `docs.page` [parameter](../writing-stories/parameters.md#global-parameters) in [`.storybook/preview.js`](../configure/overview.md#configure-story-rendering).
+Write a custom React component and update the `docs.page` [parameter](../writing-stories/parameters.md) to render the custom documentation.
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/storybook-preview-disable-docspage.js.mdx',
+    'common/custom-docs-page.js-component.js.mdx',
+    'common/custom-docs-page.ts-component.ts.mdx',
+    'common/button-story-docspage-with-custom-component.js.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
+<div class="aside">
```

Comment:
@jonniebigodes could you share an example for this? It's not obvious what needs to be done.

Reply:
@winkerVSbecks just pushed a commit to expand this section and here's the [reproduction repository](https://github.com/jonniebigodes/angular-test-custom-doc) (branch name is custom_component) with what was done in terms of changes.

Let me know what you think and we'll go from there.